### PR TITLE
Hotfix/landing desktop

### PR DESCRIPTION
**Pull request by herecomesjaycee**
This is my suggested changes to the landing page. 

The last thing I've noticed is that in desktop view (width > 1000px), the geo logo is out of of the div. Once that's addressed I can't think of anything to point out, worth asking Ajara to give comments for any styling fixes that we missed.

### CODE REVIEW

**Comment by GenesisSelf** [comment]
Yes I noticed the geo logo out of place. It's bizarre because the wrapper container is very low below the elements.

File: src/App.css [hunk 1]
```diff
@@ -471,15 +469,6 @@ Footer {
   max-width: 130px;
 }
 
-@media only screen and (min-width: 720px) {
-  .logo {
```
Comment (GenesisSelf) [inline src/App.css:475]: The logo has to be able to move outside the list element. I have updated the logo sizing in my PR

File: src/App.css [hunk 2]
```diff
@@ -332,9 +332,7 @@ Footer {
   box-sizing: border-box;
   position: relative;
   background: #D81921;
-  bottom: 3rem;
   margin-top: 0rem;
-  height: 536px;
```
Comment (GenesisSelf) [inline src/App.css:337]: I noticed a duplicate list of footer links and a chunk of white space. I placed this in as a temporary fix till we have more time to go back.

**Comment by GenesisSelf** [comment]
Hi Jaycee, thanks for reviewing. Perhaps when we meet we can go through this more.

**Comment by herecomesjaycee** [comment]
Closing as the base branch will be merged without the proposed changes.

**Pull request closed without merging**